patches/server/0034-IntelliJ-Remove-Streams.patch

```diff
@@ -0,0 +1,603 @@
+--- a/src/main/java/net/minecraft/server/EnumProtocol.java
++++ b/src/main/java/net/minecraft/server/EnumProtocol.java
+@@ -65,15 +65,17 @@ public enum EnumProtocol {
+             }
+ 
+             EnumProtocol.e[k - -1] = enumprotocol;
+-            enumprotocol.h.forEach((enumprotocoldirection, enumprotocol_a) -> {
+-                enumprotocol_a.a().forEach((oclass) -> {
++            for (Map.Entry<EnumProtocolDirection, a<?>> entry : enumprotocol.h.entrySet()) {
++                EnumProtocolDirection enumprotocoldirection = entry.getKey();
++                a<?> enumprotocol_a = entry.getValue();
++                for (Class<? extends Packet<?>> oclass : enumprotocol_a.a()) {
+                     if (EnumProtocol.f.containsKey(oclass) && EnumProtocol.f.get(oclass) != enumprotocol) {
+                         throw new IllegalStateException("Packet " + oclass + " is already assigned to protocol " + EnumProtocol.f.get(oclass) + " - can't reassign to " + enumprotocol);
+                     } else {
+                         EnumProtocol.f.put(oclass, enumprotocol);
+                     }
+-                });
+-            });
++                }
++            }
+         }
+ 
+     }
+diff --git a/src/main/java/net/minecraft/server/PlayerChunkMap.java b/src/main/java/net/minecraft/server/PlayerChunkMap.java
+index 74998ea20176c2753a0ebb03833f13f3f58a186a..7702169fcf0c2cf72f19ef9472ac435cea05cd21 100644
+--- a/src/main/java/net/minecraft/server/PlayerChunkMap.java
++++ b/src/main/java/net/minecraft/server/PlayerChunkMap.java
+@@ -546,13 +546,13 @@ public class PlayerChunkMap extends IChunkLoader implements PlayerChunk.d {
+         if (viewDistance > 6) {
+             ChunkCoordIntPair front7 = player.getChunkInFront(7);
+             pos.setValues(front7.x << 4, 0, front7.z << 4);
+-            MCUtil.getSpiralOutChunks(pos, 3).forEach(coord -> {
++            for (ChunkCoordIntPair coord : MCUtil.getSpiralOutChunks(pos, 3)) {
+                 if (shouldSkipPrioritization(coord)) {
+-                    return;
++                    continue;
+                 }
+                 double dist = MCUtil.distance(playerChunkX, 0, playerChunkZ, coord.x, 0, coord.z);
+                 updateChunkPriorityMap(priorities, coord.pair(), (int) (25 - Math.sqrt(dist) * twoThirdModifier));
+-            });
++            }
+         }
+ 
+         if (priorities.isEmpty()) return;
+diff --git a/src/main/java/net/minecraft/server/Reputation.java b/src/main/java/net/minecraft/server/Reputation.java
+index dffac0dc85fe7d093c58663f91ad687a6be50ad5..5f6a4281204c60f059437e9f04725afa9ff0578d 100644
+--- a/src/main/java/net/minecraft/server/Reputation.java
++++ b/src/main/java/net/minecraft/server/Reputation.java
+@@ -100,14 +100,14 @@ public class Reputation {
+     public void a(Reputation reputation, Random random, int i) {
+         Collection<Reputation.b> collection = reputation.a(random, i);
+ 
+-        collection.forEach((reputation_b) -> {
```

Comment:
Not a stream.

https://hg.openjdk.java.net/jdk8/jdk8/jdk/file/tip/src/share/classes/java/lang/Iterable.java#l72